### PR TITLE
Nginx version bump 1.9.9 -> 1.14.0 which also fixes a Nginx/mod_zip c…

### PR DESCRIPTION
**Pull request by gregorkistler**
…ompilation warning

As figured out in #51 there are [two warnings regarding the mod_zip](https://travis-ci.org/openseedbox/openseedbox/jobs/369439939#L1544) module. Verified with older stable Nginx versions too like 1.8.1 and got the same result.
Newer ones like 1.10.3 and 1.12.2 aren´t affected. As 1.9.9 is actually pretty old I´ve bumped the version to the latest and greatest stable version which is 1.14.0 at the time of writing.

### CODE REVIEW

**Comment by xabolcs** [comment]
Last time (lot of months ago) when I tried to update `ngnix`, then not all module was compatible with it.

**Comment by xabolcs** [comment]
OK! 1.10.0 is now required! :+1:

https://github.com/evanmiller/mod_zip/commit/8754275ec568c49a10d10e5567ac8b5942dd9cd4

**Comment by xabolcs** [comment]
Ohh, this is an all-in-one PR.
All these changes are welcome, but I will split/squash them into "one commit per feature".

**Comment by gregorkistler** [comment]
My bad, actually just the first (and likely second about JDBC driver) commit was meant to be merged.
If needed I can close it and create a new one.

Others were just a try to get the README cut down a little bit and move most of it to wiki pages :)

**Comment by gregorkistler** [comment]
I´ve split both commits into separate ones and will close this one here now.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
-OpenSeedbox - Open Source Seedbox UI - http://www.openseedbox.com
+OpenSeedbox - Open Source Seedbox UI - https://www.openseedbox.com
```
Comment (xabolcs) [inline README.md]: They have Let's Encrypt certificate! Awesome! 👍

**Comment by xabolcs** [comment]
`LICENCE` changes are live too in commits c2ffde77faac7118af89e8dae148e51099c9fffe and a47028a554990e2a14c00171fd71837fc97ba809. :+1:

**Pull request closed without merging**